features/git-town-kill/supplied_branch/offline/remote_branch.feature

```diff
@@ -0,0 +1,22 @@
+Feature: git town-kill: killing a remote branch in offline mode
+
+  When offline and trying to kill a remove branch
+  I want to be notified that this operation is not possible
+  So that I know about my mistake and can do more appropriate actions instead.
+
+
+  Background:
+    Given I have a feature branch named "feature" on another machine
```

Comment:
I was thinking for this test we actually have fetched updates so we know about the branch but have not checked it out locally